package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@oat-sa/eslint-config-tao",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
**issue:** The version is handled by the [release tool](https://github.com/oat-sa/tao-extension-release), so no need to update it.

The version itself is not correct as it should be a major change since the base dependency was also upgraded to a major.

Reply:
got it, reverted